src/cli/cli.cc

```diff
@@ -144,6 +144,57 @@ String getSocketHome () {
   return socketHome;
 }
 
+String getAndroidHome() {
+  auto androidHome = getEnv("ANDROID_HOME");
+
+  if (androidHome.size() > 0)
+    return androidHome;
+
+  if (androidHome.size() == 0) {
+    androidHome = settings["android_" + platform.os + "_home"];
+  }
+
+  if (androidHome.size() == 0) {
+    androidHome = settings["android_home"];
+  }
+
+  if (androidHome.size() == 0) {
+    if (!platform.win) {
+      auto cmd = String(
+        "dirname $(dirname $(readlink $(which sdkmanager 2>/dev/null) 2>/dev/null) 2>/dev/null) 2>/dev/null"
+      );
+
+      auto r = exec(cmd);
+
+      if (r.exitCode == 0) {
+        androidHome = trim(r.output);
+      }
+    }
+  }
+
+  if (androidHome.size() == 0) {
+    if (platform.mac) {
+      androidHome = getEnv("HOME") + "/Library/Android/sdk";
+    } else if (platform.unix) {
+      androidHome = getEnv("HOME") + "/android";
+    } else if (platform.win) {
```

Comment:
I know I left this as a `TODO`, but perhaps there is a standard location where the Android SDK is on Windows like `%localappdata%\Android\Sdk` (aka `C:\Users\user\AppData\Local\Android\Sdk`)

Reply:
Next PR.